tutorials/quickstart-node.md

```diff
@@ -0,0 +1,497 @@
+# Quick Start: Node and TimescaleDB
+
+### Goal
+This quick start guide is designed to get the Node.js developer up 
+and running with TimescaleDB as their database. In this tutorial, 
+you’ll learn how to:
+
+* [Connect to TimescaleDB](#new_database)
+* [Create a relational table](#create_table)
+* [Generate a Hypertable](#create_hypertable) 
+* [Insert a batch of rows into your Timescale database](#insert_rows)
+* [Execute a query on your Timescale database](#execute_query)
+
+### Pre-requisites
+
+To complete this tutorial, you will need a cursory knowledge of the Structured Query 
+Language (SQL). The tutorial will walk you through each SQL command, but it will be 
+helpful if you've seen SQL before.
+
+To start, [install TimescaleDB][install-timescale]. Once your installation is complete, 
+we can proceed to ingesting or creating sample data and finishing the tutorial.
+
+Obviously, you will need to [install Node][node-install] and the 
+[Node Package Manager (npm)][npm-install] as well.
+
+### Connect Node to TimescaleDB [](new-database)
+
+TimescaleDB is based on PostgreSQL and we can use common PostgreSQL tools to connect 
+your Node app to the database. In this example, we will use a common Node.js
+Object Relational Mapper (ORM) called [Sequelize][sequelize-info].
+
+#### Step 1: Create your Node app
+
+Let's initialize a new Node app. From your command line, type the following:
+
+```bash
+npm init -y
+```
+
+This will create a `package.json` file in your directory, which contains all
+of the depenencies for your project:
+
+```json
+{
+  "name": "node-sample",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
+}
+```
+
+Now, let's install Express.js by running the following command:
+
+```bash
+npm install express
+```
+
+This will create a folder called `node_modules`.
```

Comment:
This isn't needed for a tutorial that isn't talking about javascript dependencies basics